Source/WTF/wtf/Logger.cpp

```diff
@@ -68,4 +68,16 @@ Vector<std::reference_wrapper<Logger::MessageHandlerObserver>>& Logger::messageH
     return observers;
 }
 
+const Logger& emptyLogger()
+{
+    static NeverDestroyed<Ref<Logger>> emptyLogger = [&] {
```

Comment:
Nit: Since `Logger` is already heap-allocated there's no benefit to using `NeverDestroyed`. We could just leak it instead.

Reply:
@aestes We discussed offline how the other benefit of using NeverDestroyed is that we can use its location as the "owner" of the `Logger`.